snuba/migrations/snuba_migrations/spans_experimental/columns.py

```diff
@@ -0,0 +1,30 @@
+from typing import Sequence
```

Comment:
Seems like it would be easy to accidentally change the columns definition in this file now in subsequent migrations - we need to be careful not to do that